pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -53,9 +53,9 @@ const (
 //              b. Notify dirty rules
 //
 type Controller struct {
-	// antreaClient provides interfaces to watch Antrea AddressGroups,
-	// AppliedToGroups, and NetworkPolicies.
-	antreaClient versioned.Interface
+	// antreaClientProvider provides interfaces to get antreaClient, which can be
+	// used to watch Antrea AddressGroups, AppliedToGroups, and NetworkPolicies.
```

Comment:
can we add some more details to the comment here. I am assuming that when the client "expires" (e.g. because the apiserver cert has been rotated), the watch will be interrupted and we will get a new client?

Reply:
Verifying server certificate only take place for new requests and existing watches won't be interrupted by rotating cert. The new client will be used after the existing watches expire.
Added the comment.